pages/linux/uvcdynctrl.md

```diff
@@ -0,0 +1,31 @@
+# uvcdynctrl
+
+> A libwebcam command line tool to manage dynamic controls in uvcvideo.
+
+- List all available cameras:
+
+`uvcdynctrl -l`
+
+- Specify the device to use (defaults to `video0`):
+
+`uvcdynctrl -d {{device_name}}`
+
+- List available controls:
+
+`uvcdynctrl -c`
+
+- Set a new control value (for negative values: `-s '{{control_name}}' -- -42`):
+
+`uvcdynctrl -s {{control_name}} {{value}}`
+
+- Retrieve the current control value:
```

Comment:
s/Retrieve/Get

Reply:
Are you suggesting changing the word Retrieve to Get ?

Reply:
Yes. This is the standard notation to indicate word substitution in code reviews.